go-controller/pkg/config/config_test.go

```diff
@@ -794,6 +798,7 @@ var _ = Describe("Config Operations", func() {
 			gomega.Expect(OVNKubernetesFeature.EgressIPNodeHealthCheckPort).To(gomega.Equal(4321))
 			gomega.Expect(OVNKubernetesFeature.EnableMultiNetwork).To(gomega.BeTrue())
 			gomega.Expect(OVNKubernetesFeature.EnableNetworkSegmentation).To(gomega.BeTrue())
+			gomega.Expect(OVNKubernetesFeature.EnablePreconfiguredUDNAddresses).To(gomega.BeTrue())
```

Comment:
I'm probably missing something - where do you define this using the CLI options ? Is this the CLI default ?

Reply:
In this test the CLI args defined below starting at line 818, the feature arg is at line 877.